backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/nudger/BpkNudger.kt

```diff
@@ -104,6 +109,69 @@ fun BpkNudger(
     }
 }
 
+@Composable
+fun BpkNudger(
+    value: Int,
+    onValueChange: (Int) -> Unit,
+    min: Int,
+    max: Int,
+    title: String,
+    modifier: Modifier = Modifier,
+    subtitle: String? = null,
+    icon: BpkIcon? = null,
+    enabled: Boolean = LocalFieldStatus.current != BpkFieldStatus.Disabled,
+) {
+    Row(
+        modifier = modifier
+            .semantics {
+                text = listOfNotNull(title, subtitle).joinToString(separator = " ").let(::AnnotatedString)
+            }
+            .nudgerSemantics(value, onValueChange, min..max, enabled),
+        verticalAlignment = Alignment.CenterVertically,
+    ) {
+        if (icon != null) {
+            BpkIcon(
+                icon = icon,
+                contentDescription = null, // handled by semantics modifier
+                modifier = Modifier
+                    .align(Alignment.Top)
+                    .padding(end = BpkSpacing.Md),
+            )
+        }
+        Column(modifier =
+            Modifier.weight(1f)
+                .padding(end = BpkSpacing.Md),
+        ) {
+            BpkText(
+                text = title,
+                style = BpkTheme.typography.heading5,
+                color = BpkTheme.colors.textPrimary,
+                maxLines = 1,
+                overflow = TextOverflow.Ellipsis,
+                modifier = Modifier.invisibleSemantic(),
```

Comment:
Otherwise the individual texts will still be selectable despite mergeDescendants=true